components/observations/uploader/ObservationsUploader.ts

```diff
@@ -197,6 +197,43 @@ export class ObservationUploader {
         });
       });
 
+      observationFormData.avalanches.forEach((avalanche, index) => {
+        avalanche.images?.forEach(({image, caption}) => {
+          tasks.push({
+            id: uuid.v4(),
+            parentId: observationTaskId,
+            avalancheId: index,
+            attemptCount: 0,
+            type: 'image',
+            status: 'pending',
+            data: {
+              apiPrefix: apiPrefix,
+              image: {
+                uri: image.uri,
+                width: image.width,
+                height: image.height,
+                exif: image.exif
+                  ? {
+                      Orientation: typeof image.exif.Orientation === 'string' || typeof image.exif.Orientation === 'number' ? image.exif.Orientation : undefined,
+                      DateTimeOriginal: typeof image.exif.DateTimeOriginal === 'string' ? image.exif.DateTimeOriginal : undefined,
+                    }
+                  : undefined,
+              },
+              caption,
+              name: name ?? '',
+              title: `Public Observation: ${avalanche.location}`,
```

Comment:
Is this title copy-pasta from the obs form? Is it still correct in this context?